ui/docs/reference/integrations/github/checks.mdx

```diff
@@ -0,0 +1,43 @@
+# Taskcluster-Github Checks
+
+Taskcluster-Github supports [Github's Checks API](https://developer.github.com/v3/checks/).
+At the moment, the feature is in beta. If you would like to try it out:
+* Make sure you use `.taskcluster.yml` v1
+* Add `reporting: checks-v1` line at the root level of your `.taskcluster.yml`
+* You will need `queue:route:checks` scope
```

Comment:
What does this mean?  Do I as a user need the scope?

Reply:
I'm not sure I entirely understand your question, but I'll try my best to answer :) So, it's the queue client who will need the scope, and the way I was imagining this, the scope has to be provided by the user in the `scopes` section of the same task that provides the artifact name.

Can you talk more about your concern and your suggestions here?

Reply:
Please add that information to the documentation.

Reply:
oh, wait, you were asking about `queue:route:checks` scope - it's the user's github client will need it. I think people need to contact releng team for the scope at this point. I will add this info there (I was hesitant to add it because today it's releng team and tomorrow and it might be somebody else)

Reply:
It's the repository role (`repo:github.com/org/repo:action`) that needs the scope.  I think that's a clearer term for it than "github client".

I agree that we should not mention releng here.

Reply:
Sounds good, I'll add that info